src/jit/morph.cpp

```diff
@@ -1571,6 +1586,8 @@ void fgArgInfo::ArgsComplete()
                 // Spill multireg struct arguments that have Assignments or Calls embedded in them
                 curArgTabEntry->needTmp = true;
             }
+#ifndef _TARGET_ARM_
+            // Not tested yet
```

Comment:
If we uses NYI, legacyJIT/arm32 will be not works. I think It's better to remain it as `TODO` comment.